### DISCUSSION TITLE
Import json

### DISCUSSION THREAD

**Discussion by siftuser** [Security-Onion-Solutions/securityonion#5493] [Unsupported Versions] [answered]
Is there a way to import sysmon events in json format & run hunts/sigma rules against them ? Thank you in advance

**Comment by defensivedepth** [comment]
There is an open issue for this functionality, though with evtx - https://github.com/Security-Onion-Solutions/securityonion/issues/2206

Does it need to be JSON or would evtx work?

**Comment by siftuser** [comment, marked as answer]
Thank you Josh.

Appreciate if it's feasible to provide facility to import evtx stored as json, as some of our tooling export the collected data as json

**Reply by defensivedepth** [reply]
Understood. Initial support only works with evtx, but will make note of this for further review.
